Library/Formula/gauge.rb

```diff
@@ -76,6 +76,11 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
+    Installing gauge with homebrew doesn't install Gauge plugins which are required to run Gauge specs.
+    To install Gauge plugin, please explicitly run `gauge --install <plugin_name>`. Language plugins supported are `java`, `csharp` and `ruby`. Report plugins available are `html-report` and `xml-report`.
```

Comment:
Hi @DomT4,

Ideally, we would like to install the plugins in `postinstall`. But we leave it upto the users to select the plugins that they require. Which means we need to have interactive homebrew formula for installing the plugins in `postinstall`. Since homebrew discourages this practice, we have put in a caveat. 

So, as suggested, we will update the pull request with the caveat text broken into multiple lines.